core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
That tightly couples the security of the Controller to having an nginx with that exact configuration running in front of it. I don't think that is desirable. Also, when you deploy a system using cert authentication, you'll always have the overhead of also checking basic auth unnecessarily.